website/docs/artifacts.md

```diff
@@ -0,0 +1,221 @@
+# Artifacts
+
+Artifacts are used to configure what actually gets installed with a package.
+Anything that needs to be installed needs an entry in the artifacts section.
+
+There are different types of artifacts which are installed to different locations
+on the target system.
+What location this is depends on the target OS/distro and the kind of artifact.
+
+
+## Artifact Configuration
+
+Most artifact types share a common data type so can be configured similarly.
+It is shown here as a reference which is linked to in the artifact descriptions
+where it is pertinent.
+
+Configuration options:
+
+- *subpath*(string): The provided path is joined to the typicall install path,
+                     e.g. `/usr/bin/<subpath>`, where the artifact will be
+                     installed to.
+- *mode*(octal): file permissions to apply to the artifact.
+
+
+### Binaries
+
+Binaries are binary files that may be executed.
+On Linux these would typically get installed into `/usr/bin`.
+
+Binaries are a mapping of file path to [artifact configuration](#artifact-configuration).
+The file path is the path to a file that must be available after the build
+section has finished.
+
+Example:
+
+```yaml
+artifacts:
+  binaries:
+    build_output/my_bin:
```

Comment:
```suggestion
    my_src/my_bin:
```
Can you change this to make it clear that generally, output binaries will be extracted from underneath source dirs?